django/contrib/auth/views.py

```diff
@@ -200,6 +201,10 @@ def password_reset(request, is_admin_site=False,
                     RemovedInDjango110Warning, 3
                 )
                 opts = dict(opts, domain_override=request.get_host())
+
+            if extra_email_context is not None:
+                opts.update({'extra_email_context': extra_email_context})
```

Comment:
`opts.update(extra_email_context)` (since `extra_email_context` will be a dictionary)